resources/todomvc/architecture-examples/react-redux/webpack.common.js

```diff
@@ -0,0 +1,36 @@
+const path = require("path");
+
+module.exports = {
+    entry: {
+        app: "./src/index.js",
+    },
+    output: {
+        filename: "[name].bundle.js",
+        path: path.resolve(__dirname, "dist"),
+        clean: true,
+    },
+    resolve: {
+        extensions: [".js", ".jsx"],
+    },
+    module: {
+        rules: [
+            {
+                test: /\.(js|jsx)$/,
+                exclude: /nodeModules/,
+                use: {
+                    loader: "babel-loader",
+                    options: {
+                        presets: [
+                            ["@babel/preset-env", { targets: "defaults" }],
+                            ["@babel/preset-react", { runtime: "automatic" }],
+                        ],
+                    },
```

Comment:
Do we need these options if we have `babel.config.js`? It looks confusing to have both.

Reply:
added a note in the babel.config.js file, which is only used for the Jest tests. 
I kept those tests since they already existed.

Reply:
fixed the babel.config file